content/en/snippets/others/delay.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Delay (with await)"
+weight: 10
+ie_support: false
+---
+
+The `delay` function that sleeps/waits for the seconds you specified.
+
+```js
+function delay(seconds) {
+  return new Promise(function (resolve) {
+    setTimeout(resolve, seconds * 1000);
+  });
+}
+
+return (async () => {
+  // 処理の解決を待つために
+  // Promise オブジェクトを返す必要があります。
+  // このスコープの中にコードを書き込んでください。
```

Comment:
It looks like this is a Japanese comment for the English page 🙂

Reply:
fixed the loc issue